codes/classical/q-ary_digits/simplex.yml

```diff
@@ -0,0 +1,27 @@
+code_id: simplex
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Simplex code'
+
+description: |
+  An \([n,k,q^{k-1}]\) linear code over \(\mathbb{F}_q\), where \(n=\frac{q^k-1}{q-1}\). The columns of the generator matrix are in one-to-one correspondence with the elements of the projective space \(PG(k-1,q)\), with each column being a chosen representative of the corresponding element.
+
+features:
+
+  decoders:
+    - 'A quantum decoding algorithm \cite{manual:{A. Barg and S. Zhou, A quantum decoding algorithm of the simplex code, 2004.}}.'
+
+
+notes:
+  - 'The non-zero codewords have a constant weight \(q^k-1\).'
+  - 'For \(q=2\), the codewords form a \((2^k-1)\)-simplex of constant edge length if the codewords are interpreted as points in \(\mathbb{R}^n\).'
+  - 'Saturates the Griesmer bound.'
+
```

Comment:
The simplex code is the dual code of the 1-error-correcting Hamming code.

Reply:
This is already included in the "cousins" section. Or do I need to put it elsewhere, or create a stub for q-ary Hamming code?

Reply:
This is more a feature of the code, so perhaps it's better to include it in the description of the code. In the "cousins" section simply write "Hamming codes" without explaining the precise relation.

Reply:
Agreed. But I'd also put a brief explanation in the cousin detail.